R/repeat_linter.R

```diff
@@ -0,0 +1,46 @@
+#' Repeat linter
+#'
+#' Check that `while` is not used for infinite loops.
```

Comment:
wording suggestion here. this could read as "any infinite while loop is detected" which is broader than the actual linter. maybe "simple infinite loops", or just mention while(TRUE) here